x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/vectors/70_dense_vector_telemetry.yml

```diff
@@ -0,0 +1,56 @@
+setup:
+  - skip:
+      features: headers
+
+  - do:
+      indices.create:
+        index: index1
+        body:
+          settings:
+            index.number_of_shards: 2
+            index.number_of_replicas: 0
+          mappings:
+            properties:
+              vector1:
+                type: dense_vector
+                dims: 768
+                index: true
+                similarity: l2_norm
+              vector2:
+                type: dense_vector
+                dims: 1024
+                index: true
+                similarity: dot_product
+              vector3:
+                type: dense_vector
+                dims: 100
+                index: false
+
+  - do:
+      indices.create:
+        index: index2
+        body:
+          settings:
+            index.number_of_shards: 2
+            index.number_of_replicas: 0
+          mappings:
+            properties:
+              vector1:
+                type: dense_vector
+                dims: 768
+                index: true
+                similarity: l2_norm
+
+---
+"Field mapping stats":
+  - skip:
+      version: " - 8.2.99"
```

Comment:
I think that the version needs updating here, as well as the version in the reason at the next line.